src/xcode/ENA/ENA/Source/Workers/Store/Store.swift

```diff
@@ -145,32 +143,14 @@ protocol StatisticsCaching: AnyObject {
 protocol PrivacyPreservingProviding: AnyObject {
 	/// A boolean storing if the user has already confirmed to collect and submit the data for PPA. By setting it, the existing anlytics data will be reset.
 	var isPrivacyPreservingAnalyticsConsentGiven: Bool { get set }
+	// Do not mix up this property with the real UserMetadata in the PPAnalyticsData protocol
+	var userdata: UserMetadata? { get set }
```

Comment:
Personal preference would be `userData` but that's just me… :)

Reply:
Can we do :)